.github/dependabot.yaml

```diff
@@ -18,9 +22,17 @@ updates:
     ignore:
       - dependency-name: github.com/google/cel-go
       - dependency-name: github.com/quic-go/*
+    groups:
+      go-modules:
```

Comment:
PR title will be `chore(caddy): Bump the go-modules group across 1 directory with X updates`